### ISSUE TITLE
Using pre-trained models for training

### ISSUE THREAD

**Issue by RanganThaya** [allenai/allennlp#2449]
**Question**

How can I use an existing pre trained model inside my model ? For instance, can I use the NER model inside a model I am building and back propagate the loss through the pre trained model as well.

**Comment by matt-gardner** [comment]
AllenNLP `Models` are pytorch `Modules`, so you should be able to instantiate and use a `Model` just as you would any other `Module`.  You just have to be careful about giving it the right inputs and extracting the right outputs, and saving the `Model` as a member variable so it gets gradients propagated correctly.
